### PR TITLE
Added visualization for all collisions and omega random force

### PR DESCRIPTION
**Pull request by tushartk**

### CODE REVIEW

File: gym_art/quadrotor_multi/quad_utils.py [hunk 1]
```diff
@@ -252,6 +252,10 @@ def perform_collision(dyn1, dyn2):
     dyn1.vel += cons_rand_val + np.random.normal(0, 0.15, 3)
     dyn2.vel += -cons_rand_val + np.random.normal(0, 0.15, 3)
 
+    cons_rand_omega = np.random.normal(0, 0.8, 3)
+    dyn1.omega += cons_rand_omega + np.random.normal(0, 0.15, 3)
+    dyn2.omega += -cons_rand_omega + np.random.normal(0, 0.15, 3)
+
```
Comment (alex-petrenko) [inline gym_art/quadrotor_multi/quad_utils.py:258]: What is this measured in? 0.8 radians per second?
I'm surprised that values for velocity and angular velocity are magically equivalent. Can you show the video?

File: gym_art/quadrotor_multi/quadrotor_multi.py [hunk 2]
```diff
@@ -225,29 +226,34 @@ def step(self, actions):
...
+        # Applying random forces for all collisions between drones
         if self.apply_collision_force:
-            for val in self.curr_collisions:
+            for val in self.curr_drone_collisions:
                 perform_collision(self.envs[val[0]].dynamics, self.envs[val[1]].dynamics)
 
         # COLLISION BETWEEN QUAD AND OBSTACLE(S)
         col_obst_quad = self.obstacles.collision_detection(pos_quads=self.pos)
         rew_col_obst_quad_raw = - np.sum(col_obst_quad, axis=0)
         rew_col_obst_quad = self.rew_coeff["quadcol_bin_obst"] * rew_col_obst_quad_raw
```
Comment (alex-petrenko) [inline gym_art/quadrotor_multi/quadrotor_multi.py:249]: There's no random force when colliding with obstacles/ground yet, right?

Comment (tushartk) [reply gym_art/quadrotor_multi/quadrotor_multi.py:249]: Will be pushing those in a day or two, yeah I forgot about those

File: gym_art/quadrotor_multi/quad_utils.py (hunk 1 above)
Comment (tushartk) [reply gym_art/quadrotor_multi/quad_utils.py:258]: Oh yeah, now that I realize, let me double confirm this. This didn't have much effect and should not impact in a major way, but yeah, let me just check it.

**Pull request merged**